### DISCUSSION TITLE
Chrome video mov content type on form

### DISCUSSION THREAD

**Discussion by dro-sh** [rpldy/react-uploady#623] [Q&A]
I would like to upload my mov file to server via multipart data. But then I upload file via chrome file in form data has `application/octet-stream` Content-Type header. But I expect `video/quicktime`. On safari no problems. Have any ideas?

P.S. this problem reproduce only on chrome for mac

**Comment by yoavniran** [comment]
hi @dro-sh 

I just tried sending an upload request from chrome on mac and I see this:

![image](https://github.com/rpldy/react-uploady/assets/1102278/f4eb0c8c-8124-4f36-a5d1-154bf0977fdd)


So it looks like the mimetype is detected correctly. 

In any case, this is an OS/browser issue so unfortunately I dont have any insight into this from the uploader perspective.
